AddOns/BasicFinancials/app/app.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "0242c240-2336-450f-9678-2f13f4ce9a6e",
+  "name": "Basic Financials",
+  "publisher": "Microsoft",
+  "brief":  "Basic Financial Extensions aligns the User Experience to the license limitations",
```

Comment:
How about:

The Basic Financials extension enables a limited subset of the capabilities of Dynamics 365 Business Central.